README.md

```diff
@@ -39,28 +42,59 @@ To learn more about how to contribute to the development of this provider please
 **Did you find a vulnerability?** Please refer to the [Security Policy](https://github.com/devoteamgcloud/terraform-provider-looker/security/policy) for more information.
 
 ### Makefile
+
 The make command provides an easy way to access commands for local development of the provider.
+
 - To build the provider locally, you can execute the make command at the root of the project.
+
   ```
   make build
   ```
-  *Alternatively: 'go build -o terraform-provider-looker'*
 
+  *Alternatively: 'go build -o terraform-provider-looker'*
 - To format all the go files in the project.
+
   ```
   make format
   ```
-  *Alternatively: 'gofmt -l -s -w .'*
 
+  *Alternatively: 'gofmt -l -s -w .'*
 - To generate documentation using the tfplugindocs plugin.
+
   ```
   make docs
   ```
+
   *Alternatively: 'go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs'*
+
+
+### Use locally built provider
+
+#### Linux
```

Comment:
This method actually works on every platform, you can remove the "Linux" mention.